### DISCUSSION TITLE
hkknx 3.0.0 auf QNAP

### DISCUSSION THREAD

**Discussion by dengler5000** [brutella/hkknx-public#340] [General]
Guten Morgen, ich habe mir einen QNAP TS-435XeU-4G bestellt und würde darauf gerne mein hkknx laufen lassen. Was ist der beste Weg dafür? Docker?
Grüße

**Comment by brutella** [comment]
Welches Betriebssystem läuft auf dieser Hardware?

**Reply by dengler5000** [reply]
das ist ein QNAP-eigenes Betriebssystem und nennt sich QTS 5.2. Prozessor ist ein ARMv8 Cortex. Hilft dir das weiter?

**Reply by brutella** [reply]
Wenn du Docker-Images installieren kannst, wird das wohl die beste Wahl sein. Ich arbeite zurzeit daran, ein offizielles Docker-Images bereitzustellen.

**Reply by dengler5000** [reply]
Ja das klappt. Okay das wäre super. Hast du da einen groben Zeithorizont wann das fertig sein kann?

**Reply by brutella** [reply]
Offizielles Docker Image ist jetzt verfügbar.
https://hub.docker.com/repository/docker/brutella/hkknx

**Reply by brutella** [reply]
Konntest du das Docker-Image auf deinem System installieren?
